wg-architecture/charter.md

```diff
@@ -0,0 +1,32 @@
+# WG Architecture Charter
```

Comment:
if the group is going to be long-standing, people familiar with kube are probably more used to this being called a sig.

Reply:
changes to sig